src/main/java/thePackmaster/cards/gemspack/CleanseGem.java

```diff
@@ -0,0 +1,31 @@
+package thePackmaster.cards.gemspack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.gemspack.ReduceDebuffsAction;
+import thePackmaster.cardmodifiers.gemspack.CleanseGemMod;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class CleanseGem extends AbstractGemsCard {
+    public final static String ID = makeID("CleanseGem");
+
+    public CleanseGem() {
+        super(ID, -2, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
```

Comment:
-2 removes the energy orb on the card but they're still playable if you don#t override canUse

This may have bad interactions with other cards, is there a reason they can't be 0 cost?